File: problemas/1000/1009.js
const entrada = require('fs').readFileSync('/dev/stdin', 'utf-8').split('\n');

entrada.shift();
const [salario, vendas] = entrada.map(x => Number(x));
console.log(`TOTAL = R$ ${(salario + vendas * 0.15).toFixed(2)}`);

File: problemas/1000/1009.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) throws IOException {
        var sc = new Scanner(System.in);
        sc.nextLine();
        double salario = sc.nextDouble();
        double vendas = sc.nextDouble();
        System.out.printf("TOTAL = R$ %.2f\n", salario + vendas * 0.15);
    }
}